Source/WebKit/UIProcess/API/APIPageConfiguration.h

```diff
@@ -407,8 +445,9 @@ class PageConfiguration : public ObjectImpl<Object::Type::PageConfiguration> {
         WeakPtr<WebKit::WebExtensionController> weakWebExtensionController;
 #endif
         RefPtr<WebKit::WebPageGroup> pageGroup;
-        RefPtr<WebKit::WebPageProxy> relatedPage;
```

Comment:
Isn't this a potentially somewhat large and API-observable behavior change?

Reply:
No, WKWebViewConfiguration already has it as a WeakObjCPtr, so making API::PageConfiguration's relatedPage a WeakPtr is preserving API behavior, though only indirectly observable.  This only changes the behavior of WKPageConfigurationGetRelatedPage to match WKWebViewConfiguration, and that SPI has no users.